pysparkling/context.py

```diff
@@ -19,6 +20,8 @@
 
 log = logging.getLogger(__name__)
 
+__all__ = ['SparkContext']
```

Comment:
It is a rather big change in pysparkling API which we may want to discuss, it may not be aligned with the previous vision that were the API was on purpose different as pyspark (namely here because it is not a SparkContext that is created, it's a pysparkling context)

Reply:
Ok, needs to be discussed than. For me, I want it to be a drop-in replacement for pyspark (during tests). As such it would need to be 100% aligned with the pyspark API.

@svenkreiss , @tools4origins : what is your view on this?